e2e/test/scenarios/actions/reproductions/31587-bottom-corner-radius-action-buttons.cy.spec.js

```diff
@@ -0,0 +1,70 @@
+import {
+  editDashboard,
+  restore,
+  saveDashboard,
+  setActionsEnabledForDB,
+  visitDashboard,
+} from "e2e/support/helpers";
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+
+const sizes = [
```

Comment:
We can be specific about what sizes these are

```suggestion
const viewports = [
```